data_steward/tools/create_tier.py

```diff
@@ -15,6 +16,7 @@
 
 TIER_LIST = ['controlled', 'registered']
 DEID_STAGE_LIST = ['deid', 'base', 'clean']
+BASE = 'base'
```

Comment:
@dcampbell-vumc You need to fetch the latest changes and rebase, the values in `DEID_STAGE_LIST` are stale.